### DISCUSSION TITLE
[help] use_targets from VSCode opens _targets.R in Notepad with open = TRUE

### DISCUSSION THREAD

**Discussion by kendonB** [ropensci/targets#1137] [Help]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

reprex:
Open VSCode in a project
Open R
Run
```
use_targets()
```
`_targets.R` opens in Notepad, but it should open in the VSCode instance running the interactive R session

**Comment by wlandau** [comment]
use_targets() delegates to usethis::edit_file() for this, so it's a {usethis} issue and not a {targets} issue. There may be a VSCode option you can set to control the mechanism {usethis} uses.
